packages/PowerGrid/src/sparklines/base.ts

```diff
@@ -0,0 +1,23 @@
+import * as DG from 'datagrok-api/dg';
```

Comment:
If this file is supposed to contain common utilities, we usually call it "utils.ts" or "shared.ts"; I have nothing against "base.ts", in fact I like short names and it's a good one, but it's probably better to stick with the existing names